router.go

```diff
@@ -59,6 +60,11 @@ func (router *routerSocket) Type() SocketType {
 	return router.sck.Type()
 }
 
+// Addr returns the listen address
```

Comment:
please adapt with whatever updated `zmq4.Socket.Addr` documentation you came up with.